tests/Unit/Webhook/Service/UpdateLineItemsServiceTest.php

```diff
@@ -0,0 +1,250 @@
+    {
+        $this->lineItemRepository->expects($this->never())->method('findBy');
+        $this->lineItemRepository->expects($this->never())->method('save');
+
+        $updateLineItemCollection = new UpdateLineItemCollection(
+            new UpdateLineItemDto(
+                '52a3de8dd0f270fd193f9f4bff05232f',
+                'Unknown',
+                'https://tao.instance/ontologies/tao.rdf#i5fb5',
+                (new DateTimeImmutable())->setTimestamp(1565602371),
+                'qti-interactions-delivery'
+            )
+        );
+
+        $result = $this->sut->handleUpdates($updateLineItemCollection);
+
+        /** @var UpdateLineItemDto $updateLineItemDto */
+        $updateLineItemDto = $result->getIterator()[0];
+
+        $this->assertSame('ignored', $updateLineItemDto->getStatus());
+    }
+
+    public function testItAcceptsUpdate(): void
+    {
+        $this->lineItemRepository->expects($this->once())
+            ->method('findBy')
+            ->with(
+                [
+                    'slug' => ['qti-interactions-delivery']
+                ]
+            )
+            ->willReturn(
+                [
+                    (new LineItem())
+                        ->setSlug('qti-interactions-delivery')
+                ]
+            );
+
+        $this->lineItemRepository->expects($this->once())->method('save');
+
+        $this->logger->expects($this->once())
+            ->method('info')
+            ->with(
+                'The line item id 0 was updated',
+                [
+                    'uri' => 'https://tao.instance/ontologies/tao.rdf#i5fb5'
+                ]
+            );
+
+        $this->entityManager->expects($this->once())->method('flush');
+
+        $updateLineItemCollection = new UpdateLineItemCollection(
+            new UpdateLineItemDto(
+                '52a3de8dd0f270fd193f9f4bff05232f',
+                'RemoteDeliveryPublicationFinished',
+                'https://tao.instance/ontologies/tao.rdf#i5fb5',
+                (new DateTimeImmutable())->setTimestamp(1565602371),
+                'qti-interactions-delivery'
+            )
+        );
+
+        $result = $this->sut->handleUpdates($updateLineItemCollection);
+
+        /** @var UpdateLineItemDto $updateLineItemDto */
+        $updateLineItemDto = $result->getIterator()[0];
+
+        $this->assertSame('accepted', $updateLineItemDto->getStatus());
+    }
+
+    public function testItIgnoresDuplicatedUpdates(): void
+    {
+        $lineItem = (new LineItem())
+            ->setSlug('qti-interactions-delivery');
+
+        $this->lineItemRepository->expects($this->once())
+            ->method('findBy')
+            ->with(
+                [
+                    'slug' => ['qti-interactions-delivery','qti-interactions-delivery']
+                ]
+            )
+            ->willReturn(
+                [
+                    $lineItem
+                ]
+            );
```

Comment:
Same issue here (breaking everything into new lines)

Reply:
There is no issue here. please check my comment above. https://github.com/oat-sa/simple-roster/pull/173#discussion_r537317534